apps/demo-app/src/app/examples/date-time-picker/date-time-picker-example.component.html

```diff
@@ -0,0 +1,3 @@
+<section>
+    <entry-date-time-picker [datetimeControl]="myDate" label="Test 123"></entry-date-time-picker>
```

Comment:
Implement ControlValueAccessor to provide integration with Forms API (directives ngModel, formControl, formControlName). Input [datetimeControl] should be removed.

Reply:
Maybe https://netbasal.com/forwarding-form-controls-to-custom-control-components-in-angular-701e8406cc55

Reply:
https://github.com/enigmatry/entry-angular-building-blocks/blob/features/dynamic-form-poc/libs/entry-components/forms/controls/date-picker/date-picker.component.ts